Makefile

```diff
@@ -79,24 +80,32 @@ install-tools: $(MISSPELL)
 build:
 	$(DOCKER_COMPOSE_CMD) build
 
-.PHONY: build-and-push-dockerhub
-build-and-push-dockerhub:
-	$(DOCKER_COMPOSE_CMD) --env-file .dockerhub.env -f docker-compose.yml build
-	$(DOCKER_COMPOSE_CMD) --env-file .dockerhub.env -f docker-compose.yml push
-
-.PHONY: build-and-push-ghcr
-build-and-push-ghcr:
-	$(DOCKER_COMPOSE_CMD) --env-file .ghcr.env -f docker-compose.yml build
-	$(DOCKER_COMPOSE_CMD) --env-file .ghcr.env -f docker-compose.yml push
-
-.PHONY: build-env-file
-build-env-file:
-	cp .env .dockerhub.env
-	sed -i '/IMAGE_VERSION=.*/c\IMAGE_VERSION=${RELEASE_VERSION}' .dockerhub.env
-	sed -i '/IMAGE_NAME=.*/c\IMAGE_NAME=${DOCKERHUB_REPO}' .dockerhub.env
-	cp .env .ghcr.env
-	sed -i '/IMAGE_VERSION=.*/c\IMAGE_VERSION=${RELEASE_VERSION}' .ghcr.env
-	sed -i '/IMAGE_NAME=.*/c\IMAGE_NAME=${GHCR_REPO}' .ghcr.env
+.PHONY: build-and-push
+build-and-push:
+	$(DOCKER_COMPOSE_CMD) $(DOCKER_COMPOSE_ENV) build --push
+
+# Create multiplatform builder for buildx
+.PHONY: create-multiplatform-builder
+create-multiplatform-builder:
+ifeq ($(OS), Darwin)
+	docker buildx create --name otel-builder --bootstrap --use --driver docker-container --buildkitd-config ./buildkitd.toml
```

Comment:
When running `make create-multiplatform-builder` on Mac, I got the following error:

```
docker buildx create --name otel-builder --bootstrap --use --driver docker-container --buildkitd-config ./buildkitd.toml
unknown flag: --buildkitd-config
See 'docker buildx create --help'.
make: *** [create-multiplatform-builder] Error 125
```

And when checking `--help` it seems that the `--buildkitd-config` doesn't exist:

```
❯ docker buildx create --help

Usage:  docker buildx create [OPTIONS] [CONTEXT|ENDPOINT]

Create a new builder instance

Options:
      --append                   Append a node to builder instead of changing it
      --bootstrap                Boot builder after creation
      --buildkitd-flags string   Flags for buildkitd daemon
      --config string            BuildKit config file
      --driver string            Driver to use (available: "cloud", "docker-container", "kubernetes", "remote")
      --driver-opt stringArray   Options for the driver
      --leave                    Remove a node from builder instead of changing it
      --name string              Builder instance name
      --node string              Create/modify node with given name
      --platform stringArray     Fixed platforms for current node
      --use                      Set the current builder instance
```

The command with `--config` worked fine.

I'm running on M1, macOS 14.7.1, docker version:

```
docker version
Client:
 Cloud integration: v1.0.35+desktop.5
 Version:           24.0.7
 API version:       1.43
 Go version:        go1.20.10
 Git commit:        afdd53b
 Built:             Thu Oct 26 09:04:20 2023
 OS/Arch:           darwin/arm64
 Context:           desktop-linux

Server: Docker Desktop 4.26.1 (131620)
 Engine:
  Version:          master
  API version:      1.44 (minimum version 1.12)
  Go version:       go1.21.3
  Git commit:       54fcd40aa4de94cd75aedc5f6ebf38c6d8f92082
  Built:            Wed Nov 22 07:43:39 2023
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.25
  GitCommit:        d8f198a4ed8892c764191ef7b3b06d8a2eeb5c7f
 runc:
  Version:          1.1.10
  GitCommit:        v1.1.10-0-g18a0cb0
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Reply:
I changed this to use the `--config` option, which works on MacOs and Linux (I don't have a way to confirm Windows)